packages/client/src/views/GrantDetailsView.vue

```diff
@@ -378,6 +401,9 @@ export default {
     newTerminologyEnabled() {
       return newTerminologyEnabled();
     },
+    shareTerminologyEnabled() {
+      return shareTerminologyEnabled();
+    },
```

Comment:
(very minor/optional/non-blocking) This can be simplified to `shareTerminologyEnabled,` which is a bit more terse and makes it exceedingly clear that we're not wrapping the helper method to change its functionality in any way.

Reply:
Not quite sure, what you mean here; does this mean to change this to appear like:

```
   newTerminologyEnabled() {
      return newTerminologyEnabled();
    },
    shareTerminologyEnabled,
```

?

Reply:
Ah, sorry, I think the suggestion became so terse that it lost its context and was a bit tough to understand. To be more clear, you can apply this diff and the behavior will be identical: 

```diff
diff --git a/packages/client/src/views/GrantDetailsView.vue b/packages/client/src/views/GrantDetailsView.vue
index afc16fbb..623178c9 100644
--- a/packages/client/src/views/GrantDetailsView.vue
+++ b/packages/client/src/views/GrantDetailsView.vue
@@ -404,12 +404,8 @@ export default {
         (interested) => interested.agency_id.toString() === this.selectedAgencyId,
       );
     },
-    newTerminologyEnabled() {
-      return newTerminologyEnabled();
-    },
-    shareTerminologyEnabled() {
-      return shareTerminologyEnabled();
-    },
+    newTerminologyEnabled,
+    shareTerminologyEnabled,
     unassignedAgencies() {
       return this.agencies.filter(
         (agency) => !this.assignedAgencies.map((assigned) => assigned.id).includes(agency.id),
```

Again, totally optional and non-blocking!

Reply:
Done!